dataset/companies/FNB.yml

```diff
@@ -0,0 +1,8 @@
+ - Name: "McDonalds"
```

Comment:
Please add more description to the PR's description.
Also, what is `fnb`? Let's not use shortcuts or aliases. Not everyone knows what is `FNB`